hasjob/templates/macros.html.jinja2

```diff
@@ -216,3 +237,69 @@
     </script>
   {%- endwith %}
 {%- endmacro -%}
+
+{%- macro conversion_funnel_setup() -%}
+  <script type="text/javascript">
+    window.Hasjob.Util = {
```

Comment:
Maybe add the functions under a different namespace, say, `window.Hasjob.FunnelStats`? `Util` is better reserved for functions that are generic utility functions.

Reply:
Changed